iOS/Objective C/LevelPlayDemo/DemoViewController.m

```diff
@@ -251,7 +254,10 @@ - (void)setEnablementForButton:(ButtonIdentifiers)buttonIdentifier
         UIButton *buttonToModify;
         
         switch (buttonIdentifier) {
-            case ShowRewardedVideoButtonIdentifier:
```

Comment:
this is more a network issue, please take with Noy. When clicking a banner, seeing a message "app not available - this app is currently not available in your country or region" doesn't look good

Reply:
please create a ticket